lib/b_main.ml

```diff
@@ -878,10 +878,11 @@ let event_loop anim new_anim board =
   let e = !Trigger.my_event in
   continue e 0
 
+let nop_event_fps = Time.make_fps ()
```

Comment:
this function should be renewed at each `run`, otherwise the `start` hidden variable becomes meaningless